lib/DTLZ/benchmark.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from deephyper_benchmark import *
+
+DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
```python
import os
DIR = ...
```
can be removed as they are not being used.

Reply:
removing